src/css_table.rs

```diff
@@ -61,7 +61,7 @@ impl<'a> ClassedTableGenerator<'a> {
         for (i, line) in LinesWithEndings::from(self.code).enumerate() {
             open_row(&mut self.html, i);
             if self.max_line_len.map_or(false, |n| line.len() > n) {
-                self.html.push_str(line);
```

Comment:
this line was the issue. The rest of the changes are just making it more readable